programs/token/rpc.go

```diff
@@ -28,7 +28,6 @@ import (
 const MINT_SIZE = 82
 
 func (mint *Mint) Decode(data []byte) error {
-	mint = new(Mint)
```

Comment:
why this?

Reply:
This is a bug that token mint decode method doesn't work, since it created a new pointer to a new address, leaving the older mint unchanged.  After submitting the PR I saw another PR which fixed this bug as well. #285